tests/python_client/utils/api_request.py

```diff
@@ -62,3 +52,50 @@ def api_request(_list, **kwargs):
                 log.debug("(api_request)  : [%s] args: %s, kwargs: %s" % (func.__qualname__, log_arg, str(kwargs)))
             return func(*arg, **kwargs)
     return False, False
+
+
+def logger_interceptor():
+    def wrapper(func):
+        def log_request(*arg, **kwargs):
+            arg = arg[1:]
+            arg_str = str(arg)
+            log_arg = arg_str[0:log_row_length] + '......' if len(arg_str) > log_row_length else arg_str
+            if kwargs.get("enable_traceback", True):
+                log.debug("(api_request)  : [%s] args: %s, kwargs: %s" % (func.__name__, log_arg, str(kwargs)))
+
+        def log_response(res, **kwargs):
+            if kwargs.get("enable_traceback", True):
+                res_str = str(res)
+                log_res = res_str[0:log_row_length] + '......' if len(res_str) > log_row_length else res_str
+                log.debug("(api_response) : [%s] %s " % (func.__name__, log_res))
+            return res, True
+
+        async def handler(*args, **kwargs):
+            _kwargs = copy.deepcopy(kwargs)
+            _kwargs.pop("enable_traceback", None)
+            check_task = kwargs.get("check_task", None)
+            check_items = kwargs.get("check_items", None)
+            try:
+                # log request
+                log_request(*args, **_kwargs)
+                # exec func
+                res = await func(*args, **_kwargs)
+                # log response
+                log_response(res, **_kwargs)
+                # check_response
+                check_res = ResponseChecker(res, sys._getframe().f_code.co_name, check_task, check_items, True).run()
```

Comment:
1. check func can be added to `finally:`
2. the check method does not pass in the request parameter